src/utils/helpers.ts

```diff
@@ -0,0 +1,19 @@
+import { Filter } from '../types/Filter';
+import { Todo } from '../types/Todo';
+
+export const filterTodos = (todos: Todo[], filterBy: Filter) => {
+  const filteredTodos = todos.filter(todo => {
+    switch (filterBy) {
+      case Filter.ALL:
+        return todo;
+      case Filter.ACTIVE:
+        return todo.completed === false;
+      case Filter.COMPLETED:
+        return todo.completed === true;
+      default:
+        return false;
```

Comment:
you can 'connect' it with Filter.ALLL case and use 
```suggestion
        return true;
```